src/reflect/scala/reflect/internal/Mirrors.scala

```diff
@@ -45,10 +45,6 @@ trait Mirrors extends api.Mirrors {
       else definitions.findNamedMember(segs.tail, RootClass.info member segs.head)
     }
 
-    /** Todo: organize similar to mkStatic in scala.reflect.Base */
-    private def getModuleOrClass(path: Name, len: Int): Symbol =
-      getModuleOrClass(path.toString, len, path.newName(_))
-
```

Comment:
currently `@annotation.unused` because you never know. (YNK instead of YAGNI.) (YNK pronounced eunuch.)